lib/githubDispatcher.js

```diff
@@ -21,8 +21,11 @@ module.exports = function(argv, callback) {
         password : argv.couchdbPassword
       }
     }, argv.campaignDb);
-
-  campaignModel.once('ready', issueFetchAndAdd.bind(null, 1, callback));
+  if(argv.syncFromGithub === 'yes'){
```

Comment:
I would add a space between `){`, and around the `else` below to be consistent with the style of the rest of the file.